Tests/KituraWebSocketTests/TestWebSocketService.swift

```diff
@@ -87,8 +88,11 @@ class TestWebSocketService: WebSocketService {
         }
     }
 
+    var disconnectClientId = [String]()
     public func disconnected(connection: WebSocketConnection, reason: WebSocketCloseReasonCode) {
-        XCTAssertEqual(connectionId, connection.id, "Client ID from connect wasn't client ID from disconnect")
+        disconnectClientId.append(connection.id)
+        while disconnectClientId.count != connectClientId.count {}
```

Comment:
This needs to be moved to a new test. I'd suggest we write a new test that creates an arbitrary number of connections, saving the connection ids in an array,  and then disconnects them, again saving the connection ids in another array. We should then compare the arrays.